docs/migration-guide.md

```diff
@@ -6,7 +6,29 @@ If you are starting from scratch, see the [new project guide](setting-up-a-gucdk
 
 ---
 
-## Account readiness
+## Overall migration process
+
+1. Identify the [GuCDK pattern](https://guardian.github.io/cdk/modules/patterns.html) which works for your app
+1. Check account readiness
```

Comment:
Hopefully most accounts are ready by now, so I'm not sure if we still need this step?

Reply:
That's a great point, if we could confirm that (and maybe set up the one's missing?) it'd be one less thing in the guide!

Reply:
Yeah, from what I can tell `account-readiness` checks for the existence of the parameters defined here https://github.com/guardian/cdk/blob/6caaf42ab406de16b990612a551fdfe1741cbf82/src/constants/ssm-parameter-paths.ts . I'm not sure if this is now enforced for all accounts defined in aws-account-setup, or if there's an easy way to check which accounts are missing some parameters.

Reply:
Yeah `guardian/aws-account-setup` is aimed at getting (and keeping) AWS accounts GuCDK-ready. I think we can remove this step from the GuCDK docs. Indeed, we might be able to remove this from the GuCDK CLI (this might need a bit more thinking about though).

Maybe the [AWS Account Information dashboard](https://metrics.gutools.co.uk/d/de7dtpiflscn4a/aws-account-information) could be expanded with a "GuCDK-ready" box?

Reply:
Thanks both; I've removed this in https://github.com/guardian/cdk/pull/2743/commits/c8288fe6dd4d8166e5abd4fb017504b8974735b9.